.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -0,0 +1,28 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on PR
+'on': pull_request
+jobs:
+  build_and_preview:
+    if: '${{ github.event.pull_request.head.repo.full_name == github.repository }}'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build site
+        env:
+          ELEVENTY_ENV: prod
```

Comment:
We may want to leave this as a dev build because it will be faster and it will show `draft` posts. This was something we ran into recently while working on the Learn CSS course.